### PR TITLE
Changed default font family to Zilla Slab

### LINKED ISSUE #114: Update default font-family to Zilla Slab

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #114 

/cc @kristinashu @alanmoo @gvn 

screen shot 2017-08-11 at 13 26 18-fullpage

### CODE REVIEW

File: src/scss/overrides/_variables.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@ html {
 $list-item-vertical-margin: 0.25em;
 
 
-$font-family-sans-serif: 'Fira Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-sans-serif: 'Zilla Slab', 'Open Sans', X-LocaleSpecific, sans-serif;
```
Comment (mmmavis) [inline src/scss/overrides/_variables.scss]: I followed how mozilla.org is doing it. Does this look like a reasonable fallback order? @kristinashu

File: src/index.pug [hunk 2]
```diff
@@ -3,7 +3,7 @@ html
   head
     meta(charset='utf-8')
     meta(name='viewport', content='width=device-width, initial-scale=1, shrink-to-fit=no')
-    link(rel='stylesheet', href='//code.cdn.mozilla.net/fonts/fira.css')
+    link(rel='stylesheet', href='//code.cdn.mozilla.net/fonts/zilla-slab.css')
     link(rel='stylesheet', href='css/mofo-bootstrap.css')
```
Comment (mmmavis) [inline src/index.pug]: `css/mofo-bootstrap.css` was a 404 so I made changes to `package.json` to copy this file over to `demo/css/`

**Comment by kristinashu** [comment]
Zilla looks great! The only thing is that "UI elements" should be in open sans. For this, I think that only effects the button text.

**Comment by mmmavis** [comment]
kk will update font type for buttons

**Comment by gvn** [comment]
Assuming design looks good the code looks good!

**Pull request merged**